rqt_bag/src/rqt_bag/bag_widget.py

```diff
@@ -330,7 +345,8 @@ def _set_status_text(self, text):
             self.progress_bar.setTextVisible(False)
 
     def _update_status_bar(self):
-        if self._timeline._timeline_frame.playhead is None or self._timeline._timeline_frame.start_stamp is None:
+        if (self._timeline._timeline_frame.playhead is None or \
```

Comment:
In cases where parentheses are used, we don't need the backslash for line continuation.
```suggestion
        if (self._timeline._timeline_frame.playhead is None or
```